reference-implementation/__tests__/resolving.js

```diff
@@ -153,8 +153,8 @@ describe('Mapped using the "imports" key only (no scopes)', () => {
         "/lib/no.mjs": null,
         "./dotrelative/no.mjs": [],
 
-        "/": "/lib/slash-only.mjs",
-        "./": "/lib/dotslash-only.mjs",
+        "/": "/lib/slash-only.mjs/",
+        "./": "/lib/dotslash-only.mjs/",
```

Comment:
These are URL-like, so shouldn't have this requirement, I think. I will try to fix.

Reply:
I guess it comes down to https://github.com/domenic/import-maps/issues/91 though, so, maybe I won't do anything...